ansible/rancher/default-ha/QUICKSTART.md

```diff
@@ -0,0 +1,43 @@
+# Quickstart
+
+## Prerequisites
+
+1. Cluster Ready: You have a k3s or rke2 cluster.
+
+## Steps
+
+### Step 1. Create Config
+
+Create `vars.yaml` in this folder with your desired settings.
+
+**Important: `fqdn` must resolve to your Load Balancer or Master Node IP (same as the `FQDN` you used for the K8s cluster).**
```

Comment:
nit; I feel like this could be more clear. i.e. 
```
prerequisites:
* k8s cluster
  * should be active
  * should resolve to an fqdn

step 1
* create vars.yaml that contains your fqdn
...
```